contracts/vendor/starkware-libs/starkgate-contracts/contracts/starkware/starknet/std_contracts/ERC20/permitted.cairo

```diff
@@ -0,0 +1,39 @@
+%lang starknet
```

Comment:
We tried to do submodule for starkgate L1 contracts and we got some issue with it so That’s why we create vendor file for now. Do you want me to do submodule for cairo contracts or do we keep the vendor as it is and add the Cairo file ?